### PR TITLE
chore(secrets): add more tests for partial vs full runs

### PR DESCRIPTION
**Pull request by omryMen**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/secrets/git_history/test_utils.py [hunk 1]
```diff
@@ -7,28 +7,6 @@
 from checkov.secrets.git_types import Commit, CommitMetadata
 
 
-# def t(a):
```
Comment (marynaKK) [inline tests/secrets/git_history/test_utils.py:10]: 😨 oh

File: checkov/main.py [hunk 2]
```diff
@@ -287,8 +287,8 @@ def run(self, banner: str = checkov_banner, tool: str = checkov_tool, source_typ
                 repo_root_for_plan_enrichment=self.config.repo_root_for_plan_enrichment,
                 resource_attr_to_omit=self.config.mask,
                 # TODO modify the output for git_history secret and remove the rewrite of enable_git_history_secret_scan
-                # enable_git_history_secret_scan=self.config.scan_secrets_history,
-                enable_git_history_secret_scan=False,  # expose after unite git history with secret scan
+                enable_git_history_secret_scan=self.config.scan_secrets_history,
+                # enable_git_history_secret_scan=False,  # expose after unite git history with secret scan
```
Comment (lirshindalman) [inline checkov/main.py:291]: Do we want to open it? I think we need to open it only after Maryna's test

**Review by marynaKK — APPROVED** [review]

**Review by lirshindalman — COMMENTED** [review]
nice!:grin: 
just one comment

File: checkov/main.py (hunk 2 above)
Comment (omryMen) [reply checkov/main.py:291]: damn.. pushed by accident again

**Review by lirshindalman — APPROVED** [review]

**Pull request merged**